docs/user-guide/disassembler-backends/ghidra.md

```diff
@@ -2,30 +2,39 @@
 
 ## Install
 
-### Docker
-If building an OFRAK Docker image, Ghidra will be automatically installed if the `disassemblers/ofrak_ghidra` package is included in the Docker build's config file.
-For example, `ofrak-ghidra.yml`:
-
-```yaml
-registry: "redballoonsecurity/ofrak"
-base_image_name: "ghidra-base"
-image_name: "ghidra"
-packages_paths:
-  [
-    "ofrak_type",
-    "ofrak_io",
-    "ofrak_patch_maker",
-    "ofrak_core",
-    "disassemblers/ofrak_ghidra",
-    "frontend",
-  ]
-entrypoint: |
-    nginx \
-      & python3 -m ofrak_ghidra.server start \
-      & python3 -m ofrak gui -H 0.0.0.0 -p 8877
-
-
-```
+=== "Docker"
+
+    If building an OFRAK Docker image, Ghidra will be automatically installed if the `disassemblers/ofrak_ghidra` package is included in the Docker build's config file.
+    For example, `ofrak-ghidra.yml`:
+
+    ```yaml
+    registry: "redballoonsecurity/ofrak"
+    base_image_name: "ghidra-base"
+    image_name: "ghidra"
+    packages_paths:
+      [
+        "ofrak_type",
+        "ofrak_io",
+        "ofrak_patch_maker",
+        "ofrak_core",
+        "disassemblers/ofrak_ghidra",
+        "frontend",
+      ]
+    entrypoint: |
+        nginx \
+          & python3 -m ofrak_ghidra.server start \
+          & python3 -m ofrak gui -H 0.0.0.0 -p 8877
+    ```
+=== "Native"
+
+    If you would like to run OFRAK natively with a local Ghidra installation, follow these steps:
```

Comment:
Isn't the first step to install `ofrak-ghidra`? For now, this should be done by running `make install` instide of the directory (until PyPI is updated).

Reply:
Ideally, our native-install instructions should work whether `ofrak-ghidra` is being installed from source or from PyPI. (Currently, the server.conf file breaks this rule, but I think it's fine to ignore this for now).

Either way, installing `ofrak-ghidra` (either from PyPI, eventually, or from source) should be a step of the native installation. The README instructions you link to also need to be updated (they should probably just point to these instructions).

Reply:
@whyitfor For redirecting the README instructions, should I replace both Docker and MacOS sections with a single link to the Install section of the OFRAK Ghidra guide?